docs/src/main/asciidoc/security-oidc-web-authentication-concept.adoc.adoc

```diff
@@ -0,0 +1,1239 @@
+////
```

Comment:
double `.adoc` is here

Reply:
I'm not sure now about this restructuring. So the remove oidc web-authentication doc had this structure:
- Introduction with the nice diagram you prepared
- Tutorial/Quickstart
- Reference

Now we have 
- Tutorial in a separate doc
- Introduction with the diagram combined with the all the reference material

I'm not sure it makes it better for the users.
I'd like to propose 2 options:
1. Move the Intro with the diagram to the top of the tutorial doc - so that by the time they go to the tutorial they have a high level picture in mind
2. Restore the original doc, keep the intro there and link to the tutorial and concept

Reply:
Note security-authentication-mechanisms link to this doc but neither of these docs refer to the tutorial. IMHO finding the tutorial should be super easy. I propose to have both of these docs refer to it. This doc should offer the tutorial link at the very top and in the references IMHO

Reply:
Perhaps it is fine to keep the diagramm in the concept but the users coming to the tutorial should be asked to check it and the whole intro first, some brief intro at the top of the tutorial might be worth having as well